adapters/repos/db/repo.go

```diff
@@ -134,8 +133,11 @@ func (db *DB) StartupComplete() bool { return db.startupComplete.Load() }
 func New(logger logrus.FieldLogger, config Config,
 	remoteIndex sharding.RemoteIndexClient, nodeResolver nodeResolver,
 	remoteNodesClient sharding.RemoteNodeClient, replicaClient replica.Client,
-	promMetrics *monitoring.PrometheusMetrics,
+	promMetrics *monitoring.PrometheusMetrics, memMonitor *memwatch.Monitor,
 ) (*DB, error) {
+	if memMonitor == nil {
+		memMonitor = memwatch.NewDummyMonitor()
+	}
```

Comment:
Can we remove this part? This looks to me like a fix for tests, where we forgot to pass the `memMonitor`. Can we always explicitly create `memMonitor` and remove this part of code?